site/app/templates/admin/admin_gradeable/AdminGradeableAuto.twig

```diff
@@ -21,7 +21,7 @@
 
     <div id="student_download_view">
         <div>
-            <fieldset><legend>View and download submitted files?</legend>
+            <fieldset><legend>View and download submitted files? <span class="warning-text" id="warning"> WARNING: Students will see the grading score and TA feedback but will not be able to download the scanned and uploaded file.</span></legend>
```

Comment:
I think the id `warning` should be changed to something more descriptive, like maybe `view-files-warning`